### DISCUSSION TITLE
RegularCartesianGrid not defined

### DISCUSSION THREAD

**Discussion by travelguy95** [CliMA/Oceananigans.jl#3085] [General]
Hello. Thank you for this contribution. I am really interested in using it. 

I tried to run the example here: https://clima.github.io/OceananigansDocumentation/v0.22.0/generated/ocean_wind_mixing_and_convection/

I am getting the error: RegularCartesianGrid not defined
Is there something I need to import? Thank you! Sorry, I am a novice with Julia.

**Comment by glwagner** [comment]
Hi!

`RegularCartesianGrid` is very old syntax by a few years. You are viewing an old version of the docs. The Oceananigans version for the docs you are looking at is shown in the bottom left corner of the window:

![image](https://user-images.githubusercontent.com/15271942/234024729-723c82a2-9cd7-478b-b866-0088e65c0752.png)

Using that dropdown menu you can change the version you are viewing to the current one (0.81.0). You could also try running code on Oceananigans version 0.22.0 by installing that version using Julia's package manager --- but that may be the path to madness. Version beware!

Here's some more reading about package management:

https://pkgdocs.julialang.org/v1/managing-packages/

PS how did you find the old documentation version? Wonder if we can suppress that...

PPS I converted your issue to discussion because I don't think this is a bug (it's a feature to be able to view old versions of the docs, albeit perhaps not a useful one...)

**Reply by travelguy95** [reply]
Thank you so much for your answer! I actually have no idea how I came upon the old docs. But according to my Google history, I searched for phrases like "oceananigans tutorial" and "oceananigans course". Somehow I just clicked on it. Weird!

**Comment by navidcy** [comment]
We should not suppress older docs since somebody might be using those versions for something. We just need to be careful and looking at the same docs version as the package we are running. [This holds for all packages at any programming language ;)]

**Reply by glwagner** [reply]
Note that old docs get a warning message (this is what I mean by suppressed), so I think this is good practice.

**Comment by navidcy** [comment]
Julia keeps its docs for v0.3 even though Julia is at 1.8 now.

<img width="1154" alt="Screenshot 2023-04-26 at 9 42 48 am" src="https://user-images.githubusercontent.com/7112768/234431027-1dc96bc1-9d10-4c7e-a8ab-29f51333d153.png">

**Reply by glwagner** [reply]
Yes but they display a helpful message that the docs are outdated.

**Comment by navidcy** [comment]
@travelguy95 I’m also wondering how you found the link to the v0.22 docs. Did google return it to you?

We (the devs) should try to minimize users to be bumping onto that old docs by accident.

**Reply by travelguy95** [reply]
It is so strange. I can't figure out how I came upon v0.22 docs. But I recall Googling repeatedly for "oceananigans tutorial" and "oceananigans course" and related phrases and clicking on whatever came up.

**Reply by navidcy** [reply]
I actually did the same now and ended up on the KH example from v0.45.2!

**Reply by navidcy** [reply]
I also found this rental --> https://www.seabrookwa.com/vacation-rentals/oceananigans

**Reply by glwagner** [reply]
could be a sign that we need tutorials

**Reply by travelguy95** [reply]
If possible, I would love to see a tutorial on these simulations: https://clima.github.io/OceananigansDocumentation/stable/gallery/

So beautiful anyone can appreciate ...
